packages/keychain/src/components/connect/CreateSession.tsx

```diff
@@ -85,21 +93,6 @@ const CreateSessionLayout = ({
     }
   }, [controller, policies, maxFee, expiresAt, onConnect]);
 
-  const onSkipSession = useCallback(async () => {
-    if (!controller || !policies) return;
-    try {
-      setError(undefined);
-      setIsConnecting(true);
-
-      const processedPolicies = processPolicies(policies, true);
-      await controller.createSession(duration, processedPolicies, maxFee);
```

Comment:
This is actually the intended logic, since we always create a wildcard session in this case and enforce policy logic locally. In this case, we approve no policies so the user is always prompted to sign. We should improve the function naming to reflect that better.

Reply:
Ah I missed the `toggleOff` input, will add back with better naming